doc/classes/Input.xml

```diff
@@ -109,6 +109,23 @@
 				[b]Note:[/b] For Android, [member ProjectSettings.input_devices/sensors/enable_gyroscope] must be enabled.
 			</description>
 		</method>
+		<method name="get_joy_accelerometer" qualifiers="const">
+			<return type="Vector3" />
+			<param index="0" name="device" type="int" />
+			<description>
+				Returns the requested joypad's accelerometer sensor readings in m/s².
+				If the joypad doesn't have an accelerometer sensor or it's currently disabled, this method returns [constant Vector3.ZERO].
+				For a joypad held in front of you, the returned axes are defined as follows:
+				-X ... +X : left ... right;
+				-Y ... +Y : bottom ... top;
+				-Z ... +Z : farther ... closer.
+				For the joypad to be able to send accelerometer data, the sensor must exist on the joypad and it must be enabled using [method set_joy_accelerometer_enabled].
+				The gravity part alone can be retrieved using [method get_joy_gravity].
+				[b]Note:[/b] The gravity part may have inaccurate values if the joypad is being rotated, because Godot can't reliably retrieve it due to OS differences.
+				[b]Note:[/b] To get approximate accelerometer movement (with the gravity part excluded), use [code]Input.get_joy_accelerometer(device) - Input.get_joy_gravity(device)[/code].
+				[b]Note:[/b] Joypad motion sensors reading is only supported on Windows, Linux and macOS.
```

Comment:
```suggestion
				[b]Note:[/b] Joypad motion sensor reading is only supported on Windows, Linux, and macOS.
```
Singular here, and please use [Oxford comma](https://contributing.godotengine.org/en/latest/documentation/guidelines/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything). For all